python/runtime/cudaq/algorithms/py_translate.cpp

```diff
@@ -0,0 +1,128 @@
+/*******************************************************************************
+ * Copyright (c) 2022 - 2024 NVIDIA Corporation & Affiliates.                  *
+ * All rights reserved.                                                        *
+ *                                                                             *
+ * This source code and the accompanying materials are made available under    *
+ * the terms of the Apache License 2.0 which accompanies this distribution.    *
+ ******************************************************************************/
+#include "cudaq/Optimizer/CodeGen/OpenQASMEmitter.h"
+#include "cudaq/Optimizer/CodeGen/Pipelines.h"
+#include "cudaq/algorithms/draw.h" // TODO  translate.h
+#include "utils/OpaqueArguments.h"
+#include "mlir/Bindings/Python/PybindAdaptors.h"
+
+#include <iostream>
+#include <pybind11/complex.h>
+#include <pybind11/stl.h>
+
+namespace cudaq {
+
+void pyAltLaunchKernel(const std::string &, MlirModule, OpaqueArguments &,
+                       const std::vector<std::string> &);
+
+std::string getQIRLL(const std::string &name, MlirModule module,
```

Comment:
What is `LL`? If it is `llvm` can we just write `llvm`?

Reply:
not sure, this is a pre-existing function I used - would it be better to change it to "getQIR"?

Reply:
Renamed!